cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -54,6 +54,9 @@
 filters_file_naming_template = 'filter_{tenant_name}.yaml'
 """Naming template for filter files."""
 
+all_tenants_filter_filename = 'all_tenants_filter.yaml'
```

Comment:
I'm not sure that this file will be used by jenkins without any changes in functional tests script.